webapp/graphite/functions/params.py

```diff
@@ -2,29 +2,31 @@
 
 from graphite.errors import InputParameterError
 from graphite.render.attime import parseTimeOffset
+from graphite.logger import log
+from graphite.functions.aggfuncs import aggFuncs, aggFuncAliases
 
 
 class ParamTypes(object):
   pass
 
 
-for paramType in [
-  'aggFunc',
-  'boolean',
-  'date',
-  'float',
-  'integer',
-  'interval',
-  'intOrInterval',
-  'node',
-  'nodeOrTag',
-  'series',
-  'seriesList',
-  'seriesLists',
-  'string',
-  'tag',
-]:
-  setattr(ParamTypes, paramType, paramType)
+class ParamType(object):
+  name = 'undefined'
```

Comment:
we don't need this afaik

Reply:
True, it will always be defined by the constructor anyway